ivy/functional/frontends/torch/Tensor.py

```diff
@@ -66,7 +66,10 @@ def amax(self, dim=None, keepdim=False):
     def contiguous(self, memory_format=torch.contiguous_format):
         return self.data
 
-    # Special Methoods #
+    def max(self, dim=None, keepdim=False):
+        return torch_frontend.max(self.data, dim=dim, keepdim=keepdim)
```

Comment:
It's giving an error: `AttributeError: module 'ivy.functional.frontends.torch' has no attribute 'max'`
Make sure that the ordinary `max()` function is implemented before implementing the instance method.
You can implement that If you want. 🙂